keyboards/keychron/c3_pro/ansi/red/config.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2024 @ Keychron (https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Indication led */
+#define LED_MAC_OS_PIN A4
+#define LED_WIN_OS_PIN A5
+#define LED_OS_PIN_ON_STATE 1
+
+/* LED Matrix Driver Configuration */
+#define SNLED27351_I2C_ADDRESS_1 SNLED27351_I2C_ADDRESS_VDDIO
```

Comment:
```suggestion
#define SNLED27351_I2C_ADDRESS_1 SNLED27351_I2C_ADDRESS_GND
```

Tested, this is working where the old value was not

Reply:
Must've mixed it up when I was updating it. I'll update right away. Thx for pointing it out.